blog/2023-01-18-free-software-localization.md

```diff
@@ -0,0 +1,72 @@
+---
+slug: free-software-localization
+
+title: "Free software localization with the Tolgee platform"
+
+authors: [mchalupnikova]
+
+image: "/img/blog/free-localization/painless_localization.png"
+
+description: "With Tolgee localization platform and its own integration libraries, you can quickly and easily translate your software into multiple languages, allowing you to reach a truly global market."
+
+tags: [localization, i18n, translation]
+---
+Overall, localization is an important consideration in software development, as it allows products to be more widely adopted and used by a global market. In addition, it improves the user experience for non-native speakers, builds a positive reputation for the company, and increases its revenue and growth. Unfortunately, software localization has been a source of difficulty for developers, with the challenge of creating and managing localization strings, integrating with external libraries, and more. 
+
+![painless localization](/img/blog/free-localization/painless_localization.png)
+
+<!--truncate-->
+
+_Imagine having a single tool to handle all these tasks, making the translation process much more manageable. A tool that is available for free and which you can self-host._ **That’s when Tolgee comes into the game.** 
+
+## Tolgee platform: what is it, and how can it help with localization?
+
+Tolgee software localization platform is a **free and open-source online tool** that helps software developers localize their products for international markets. Tolgee provides a platform that makes localization easy and affordable with an **easy-to-use interface** for managing translations and supporting all languages. It is constantly being updated with new features and improvements, making it an essential tool for anyone looking to localize their software products. In addition, it provides many great features that make the app localization a piece of cake, such as automatic or machine translation. 
```

Comment:
In addition, it provides many great features that make the app localization process a breeze, such as automatic or machine translation.